client/app/components/OnboardingTopBar/OnboardingTopBar.scss

```diff
@@ -0,0 +1,84 @@
+.topbarContainer {
+  width: 100%;
+  max-width: 120em;
+  color: white;
+  padding-top: 1.2em;
+  padding-bottom: 1.2em;
+  background-image: -moz-linear-gradient( 180deg, rgb(89,179,162) 0%, rgb(89,165,179) 100%);
+  background-image: -webkit-linear-gradient( 180deg, rgb(89,179,162) 0%, rgb(89,165,179) 100%);
+  background-image: -ms-linear-gradient( 180deg, rgb(89,179,162) 0%, rgb(89,165,179) 100%);
```

Comment:
Let's not forget to include autoprefixer with postcss and remove these vendor-prefixed values.